go.mod

```diff
@@ -0,0 +1,3 @@
+module adityatelange/hugo-PaperMod
```

Comment:
Shouldn't it be 
`module github.com/adityatelange/hugo-PaperMod` ?

Reply:
Yes!